pkg/grid/ted/make-glob.hoon

```diff
@@ -0,0 +1,26 @@
+/-  spider, docket
+/+  strandio
+=,  strand=strand:spider
+^-  thread:spider
+|=  arg=vase
+=/  m  (strand ,vase)
+^-  form:m
+=+  !<([~ [=desk dir=path]] arg)
+;<  =bowl:spider  bind:m  get-bowl:strandio
+=/  home=path  /(scot %p our.bowl)/[desk]/(scot %da now.bowl)
+=+  .^(paths=(list path) %ct (weld home dir))
+=/  =glob:docket
```

Comment:
ah, this is how you make globs! i'll keep my other comment just to note that we should document this.

Reply:
ah i see your other PR has some stuff about this. still think there should be something here that gives you that info though.